### ISSUE TITLE
SEO(backlog): XML-Sitemaps – collect URLs in XML-Sitemaps

### ISSUE THREAD

**Issue by MMitzko** [serlo-org/athene2#646]
This is a big task again ;). Please make sure that you’ve got enough capacities in the Team and that there are no other important priorities before starting.

Vroni (SEO consulting) said that collecting the URLs in XML-Sitemaps is very important for SEO. But once we collect them it’s imperative to update them regularly, so we should do this task just if we can make sure that we have enough capacities.

**Comment by inyono** [comment]
We should make a list of stuff we want in the sitemap:
* Community stuff yes/no
* All subjects? Or only "non-beta" subjects?
* Users?
* ...
And then probably a script that generates the XML sitemap from the database / server API. If possible, might be a cron job that runs weekly (or whatever), but we probably still have to submit the sitemap manually I suppose?

**Comment by arekkas** [comment]
Actually, sitemaps exist already (https://de.serlo.org/sitemap/uuid.xml) but it seems like it's broken at the moment.

**Comment by inyono** [comment]
Yeah, but probably with everything, right? I'm not sure we want that.

**Comment by arekkas** [comment]
It will be found regardless by the google crawler.
